src/microbit/test/test_button.py

```diff
@@ -51,4 +51,3 @@ def test_get_presses_two(self):
         assert 2 == self.button.get_presses()
         # Presses is reset to 0 after get_presses() is called.
         assert 0 == self.button.get_presses()
-

```

Comment:
Just FYI, but you can set up [pre-commit hooks](https://pre-commit.com/) to run checks and fixes on your commits before you send them.

Reply:
I'll look into this! Ty